dspace/config/submission-forms.xml

```diff
@@ -21,6 +21,217 @@
+                    <label>Publisher</label>
+                    <style>col-sm-8</style>
+                    <input-type>onebox</input-type>
+                    <hint>Enter the name of the publisher of the previously issued instance of this dataset.</hint>
+                    <required></required>
+                </field>
+            </row>
+            <row>
+                <field>
+                    <dc-schema>dc</dc-schema>
+                    <dc-element>identifier</dc-element>
+                    <dc-qualifier>doi</dc-qualifier>
+                    <repeatable>false</repeatable>
+                    <label>DOI</label>
+                    <input-type>onebox</input-type>
+                    <hint>Enter the DOI for the previously issued instance of this dataset.</hint>
+                    <required></required>
+                </field>
+            </row>
+            <row>
+                <field>
+                    <dc-schema>dc</dc-schema>
+                    <dc-element>relation</dc-element>
+                    <dc-qualifier></dc-qualifier>
+                    <repeatable>false</repeatable>
+                    <label>Relation</label>
+                    <input-type>onebox</input-type>
+                    <hint></hint>
+                    <required></required>
+                </field>
+            </row>
+            <row>
+                <field>
+                    <dc-schema>dc</dc-schema>
+                    <dc-element>subject</dc-element>
+                    <dc-qualifier></dc-qualifier>
+                    <repeatable>true</repeatable>
+                    <label>Subject Keywords</label>
+                    <input-type>tag</input-type>
+                    <hint>Enter appropriate subject keywords or phrases.</hint>
+                    <required></required>
+                </field>
+            </row>
+            <row>
+                <field>
+                    <dc-schema>dc</dc-schema>
+                    <dc-element>date</dc-element>
+                    <dc-qualifier>projectEnd</dc-qualifier>
+                    <repeatable>false</repeatable>
+                    <label>Project End Date</label>
+                    <input-type>date</input-type>
+                    <hint>Enter dataset project end date.</hint>
+                    <required></required>
+                </field>
+            </row>
+        </form>
+        <!-- End TAMU Customization - form definition for Dataset -->
+
+        <!-- TAMU Customization - form definition for PDAC -->
+        <!-- Form which defines the metadata page/section for depositing PDAC Entities -->
+        <form name="pdacStep">
+            <row>
+                <relation-field>
+                    <relationship-type>isDatasetAssignedToPDAC</relationship-type>
+                    <search-configuration>dataset</search-configuration>
+                    <label>Dataset</label>
+                    <hint>assign a Dataset</hint>
```

Comment:
```suggestion
                    <hint>Assign a Dataset</hint>
```

@jcreel on initial submission this hint is accurate if assign the acceptable verb as the UI does not yet afford multi-select in the manage item relationship modal. Open to other suggestions.